StreamChat/StreamChat/Model/ChatRoom.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  ChatRoom.swift
+//  StreamChat
+//
+//  Created by 김동빈 on 2021/04/27.
+//
+
+import UIKit
+
+protocol ChatRoomDelegate: AnyObject {
+    func receive(message: Message)
+}
+
+class ChatRoom: NSObject {
+    var inputStream: InputStream?
+    var outputStream: OutputStream?
+    var username = ""
+    let maxReadLength = 300
+    
+    weak var delegate: ChatRoomDelegate?
+    
+    func setupNetworkCommunication() {
+        createSocketToHost(url: ChatHost.url as CFString, port: ChatHost.port)
+        
+        guard let inputStream = self.inputStream, let outputStream = self.outputStream else {
+            return
+        }
```

Comment:
이렇게 바꾸면 더 읽기 좋을 것 같아요! guard는 정말 막는다는 느낌이 강하거든요 ㅎ
```suggestion
        guard let inputStream = self.inputStream, let outputStream = self.outputStream else { return }
```